h/api/storage.py

```diff
@@ -114,15 +118,44 @@ def create_annotation(request, data):
     # annotation.updated get created.
     request.db.flush()
 
+    update_document_metadata(
+        request.db, annotation, document_meta_dicts, document_uri_dicts)
+
+    return annotation
+
+
+def update_document_metadata(session,
+                             annotation,
+                             document_meta_dicts,
+                             document_uri_dicts):
+    """
```

Comment:
I think for now `models.document` seems the right place. It's mostly calling helper functions that are already there, right?